### PR TITLE
Enabled GitHub Actions support

### PR DESCRIPTION
**Pull request by acidicMercury8**
This pull request contains the following changes:

- Enabled GitHub Actions support using Node.js action

_I agree to follow this project's Contributing Guidelines and Code of Conduct_

### CODE REVIEW

File: .github/workflows/building.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+name: Building
+
+on:
+  push:
+    branches: [ kawaii ]
```
Comment (PeyTy) [inline .github/workflows/building.yml]: @acidicMercury8 is it possible to enable for all branches?

File: .github/workflows/building.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  pull_request:
+    branches: [ kawaii ]
+
+jobs:
+  build-windows:
+    runs-on: windows-2019
+
+    strategy:
+      matrix:
+        node-version: [ 14.x ]
```
Comment (PeyTy) [inline .github/workflows/building.yml]: @acidicMercury8 may you upgrade to 17.x? Does it depend on anything?

File: .github/workflows/building.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+    - name: Run npm
+      run: |
+        npm ci
+        npm install nexe
+
+    - name: Build Hexa
+      run: |
+        node bootstrap.js hexa.json
+        ./node_modules/.bin/nexe hexa-node.js --target windows-x64-14.15.3 --name hexa -o hexa
```
Comment (PeyTy) [inline .github/workflows/building.yml:37]: @acidicMercury8 is there better than 14.x node?

File: .github/workflows/building.yml (hunk 1 above)
Comment (acidicMercury8) [reply .github/workflows/building.yml]: @PeyTy this possibility exists, it is available using _filters_

File: .github/workflows/building.yml (hunk 3 above)
Comment (acidicMercury8) [reply .github/workflows/building.yml:37]: @PeyTy official Node.js action uses LTS releases. We can use 16.x as a maximum

File: .github/workflows/building.yml (hunk 2 above)
Comment (acidicMercury8) [reply .github/workflows/building.yml]: @PeyTy official Node.js action uses LTS releases. We can use 16.x as a maximum

File: .github/workflows/building.yml (hunk 3 above)
Comment (acidicMercury8) [reply .github/workflows/building.yml:37]: @PeyTy 14.x version is available as a maximum

**Review by PeyTy — APPROVED** [review]

**Pull request merged**